tests/getAirport.test.js

```diff
@@ -7,6 +7,12 @@ test('IN', async () => {
   expect(airport).toMatchObject({ ident: 'VABB' });
 });
 
+test('TH', async () => {
```

Comment:
This test is not need. Remove it please.

Reply:
ok